packages/ceramic-cli/src/index.ts

```diff
@@ -1,8 +1,15 @@
 import CeramicDaemon from './ceramic-daemon'
 import CeramicClient from '@ceramicnetwork/ceramic-http-client'
 import program from 'commander'
+import { serializeState } from './utils'
 
 
+const PREFIX_REGEX = /^ceramic:\/\/|^\/ceramic\//
+function validateDocId (docId: string): string {
```

Comment:
It shouldn't matter here because it's not invoked frequently, but regexes are very inefficient (and hard to read), so often a more verbose method is more efficient.

Reply:
Good point. We could update this to two `.startsWtih` if we need more clarity later.